_conferences/gdg_liverpool_devfest_2016.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "GDG Devfest"
+website: https://docs.google.com/forms/d/e/1FAIpQLSfT8oMoJHyOjNGyDVE15O-TEw5I8ABeVNeJRywj7OXlBHFdqg
```

Comment:
@blundell, looks like this file is not available.

![image](https://cloud.githubusercontent.com/assets/1859636/18791402/3ecb9978-81bb-11e6-841f-6d34ce37aa5b.png)